app/Http/Controllers/TeamSpeak/Registration.php

```diff
@@ -37,10 +37,13 @@ public function getNew()
         $autoURL = 'ts3server://'.config('services.teamspeak.host').'?nickname='.$this->account->name_first.'%20';
         $autoURL .= $this->account->name_last.'&token='.$confirmation->privilege_key;
 
+        $teamspeakURL = config('teamspeak.host');
```

Comment:
I know this controller method is already horrendous, but do you think this would look better inline?

`->with('teamspeak_url', config('teamspeak.host'))`

Just to try and avoid more temp variables?

Reply:
good call, changed